src/storage/src/hummock/compactor/iterator.rs

```diff
@@ -488,6 +488,14 @@ impl HummockIterator for ConcatSstableIterator {
     fn collect_local_statistic(&self, stats: &mut StoreLocalStatistic) {
         stats.add(&self.stats)
     }
+
+    fn value_meta(&self) -> ValueMeta {
+        let iter = self.sstable_iter.as_ref().expect("no table iter");
+        ValueMeta {
+            object_id: Some(iter.sstable_info.object_id),
+            block_id: Some(iter.seek_block_idx as u64 - 1),
```

Comment:
Better add an assertion to ensure seek_block_idx > 0

Reply:
+1

Reply:
fixed